src/Tools/Microsoft.dotnet-openapi/src/Commands/BaseCommand.cs

```diff
@@ -200,44 +200,52 @@ private async Task EnsurePackagesInProjectAsync(FileInfo projectFile, CodeGenera
                 var packageId = kvp.Key;
                 var version = urlPackages != null && urlPackages.ContainsKey(packageId) ? urlPackages[packageId] : kvp.Value;
 
-                var args = new[] {
-                    "add",
-                    "package",
-                    packageId,
-                    "--version",
-                    version,
-                    "--no-restore"
-                };
-
-                var muxer = DotNetMuxer.MuxerPathOrDefault();
-                if (string.IsNullOrEmpty(muxer))
-                {
-                    throw new ArgumentException($"dotnet was not found on the path.");
-                }
+                await TryAddPackage(packageId, version, projectFile);
+            }
+        }
 
-                var startInfo = new ProcessStartInfo
-                {
-                    FileName = muxer,
-                    Arguments = string.Join(" ", args),
-                    WorkingDirectory = projectFile.Directory.FullName,
-                    RedirectStandardError = true,
-                    RedirectStandardOutput = true,
-                };
+        private async Task TryAddPackage(string packageId, string packageVersion, FileInfo projectFile)
+        {
+            var args = new[] {
+                "add",
+                "package",
+                packageId,
+                "--version",
+                packageVersion,
+                "--no-restore"
+            };
+
+            var muxer = DotNetMuxer.MuxerPathOrDefault();
+            if (string.IsNullOrEmpty(muxer))
+            {
+                throw new ArgumentException($"dotnet was not found on the path.");
+            }
+
+            var startInfo = new ProcessStartInfo
+            {
+                FileName = muxer,
+                Arguments = string.Join(" ", args),
+                WorkingDirectory = projectFile.Directory.FullName,
+                RedirectStandardError = true,
+                RedirectStandardOutput = true,
+            };
 
-                var process = Process.Start(startInfo);
+            var process = Process.Start(startInfo);
```

Comment:
I believe Process is disposable, so this should be wrapped in a using block.
I understand that this has very little effect, given that this is a CLI tool, and after executing the command it exits and cleans up all its resources.